R/html_document.R

```diff
@@ -49,9 +49,22 @@
 #'  "darkly", "readable", "spacelab", "united", "cosmo", "lumen", "paper",
 #'  "sandstone", "simplex", or "yeti"). Pass \code{NULL} for no theme (in this
 #'  case you can use the \code{css} parameter to add your own styles).
-#'@param highlight Syntax highlighting style. Supported styles include
-#'  "default", "tango", "pygments", "kate", "monochrome", "espresso", "zenburn",
-#'  "haddock", and "textmate". Pass \code{NULL} to prevent syntax highlighting.
+#'@param highlight Syntax highlight engine and style. "default" (and "textmate")
+#'  will select highlightjs as syntax highlight engine instead of Pandoc.\cr Any
```

Comment:
Why do you use `\cr` here? (instead of just a newline)

Reply:
Because newline won't create a newline in the html help page. I believe you need `\cr` to break a line. (unless you are using markdown and you leave two space ?

```r
#' @param highlight Syntax highlight engine and style. "default" (and "textmate")
#'  will select highlightjs as syntax highlight engine instead of Pandoc.
# ' Any other value
```
will lead to no break line between "Pandoc" and "Any". 

Am I missing something ?

Reply:
Can't you do this?

```
#' @param highlight Syntax highlight engine and style. "default" (and "textmate")
#'  will select highlightjs as syntax highlight engine instead of Pandoc.
#'  
#'  Any other value
```

I'm pretty sure that works in plain Rd.

Reply:
Oh yes I can do that with en empty line, but that would make several paragraphs. It does not seem to be the style in the package currently. Every params in help file are in one paragraph. 

Let's begin to do that then - It is easier to read.

Reply:
Done in 7f11dc9